src/main/java/com/aws/greengrass/logging/impl/config/PersistenceConfig.java

```diff
@@ -362,6 +362,7 @@ protected RollingFileAppender<ILoggingEvent> getAppenderForFile(Logger loggerToC
         // Check every 2.0 second for now
         logFilePolicy.setCheckIncrement(Duration.buildBySeconds(2.0));
         logFilePolicy.setCleanLogsByLastModifiedDate(true);
+        logFilePolicy.setCleanHistoryOnStart(true);
```

Comment:
I made this change at one point, but there was a reason why I removed it. I'm not entirely sure what that reason was, I think tests were failing in Nucleus. I guess you'll find out.